src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter((p) => p.sex === 'f'
+      && people.find((p1) => p1.mother === p.name))
+    : people.filter((p) => p.sex === 'f');
```

Comment:
[CODE STYLE] - If you move your shortened return statement in arrow function to a newline, use round brackets

BAD EXAMPLE:

```
const humans = people.map((a, b) =>
    a + b);

const babies = people.map(person => children 
 ? a
 : b
```
 
GOOD EXAMPLE:

```
const humans = people.reduce((a, b) => (
  a + b
), 0);
```